pkg/cmd/openshift-controller-manager/controller/image.go

```diff
@@ -47,16 +47,16 @@ func RunImageTriggerController(ctx ControllerContext) (bool, error) {
 	sources := []imagetriggercontroller.TriggerSource{
 		{
 			Resource:  schema.GroupResource{Group: "apps.openshift.io", Resource: "deploymentconfigs"},
-			Informer:  ctx.AppInformers.Apps().InternalVersion().DeploymentConfigs().Informer(),
-			Store:     ctx.AppInformers.Apps().InternalVersion().DeploymentConfigs().Informer().GetIndexer(),
+			Informer:  ctx.AppsInformers.Apps().V1().DeploymentConfigs().Informer(),
```

Comment:
@smarterclayton can you review this please? (i think switching the trigger to external version for DC should not cause any disruption, but just want to make sure).

Reply:
It shouldn’t - will try to look today.